dashboard/app/views/levels/editors/fields/_widget_view.html.haml

```diff
@@ -3,10 +3,11 @@
 #widget_view.in.collapse
   = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :widget_view, description: "Enable Widget View"}
   %p
-    If checked, the Python Lab level will be in widget view, i.e., the code editor workspace is hidden.
+    If checked, the level will be in widget view, i.e., the code editor workspace is hidden.
```

Comment:
Maybe could add, '...the code editor workspace and view mode toggle buttons are hidden'.